lib/util/git/index.spec.ts

```diff
@@ -10,24 +10,27 @@ import {
   UNKNOWN_ERROR,
 } from '../../constants/error-messages';
 import { newlineRegex, regEx } from '../regex';
+import * as _modifiedAuth from './auth';
```

Comment:
```suggestion
import * as _auth from './auth';
```
please no unnecessary renaming